types/danish-ssn/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for danish-ssn 2.1.1
```

Comment:
```suggestion
// Type definitions for danish-ssn 2.1
```
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library
DT drops patch version, read details. Otherwise your header is ok, so I'll do comment on config details